maps/scrape.py

```diff
@@ -0,0 +1,35 @@
+import requests
+from datetime import datetime, timedelta
+import sqlite3
+
+from maps.config import *
+
+
+def format_date(dt: datetime):
+    return dt.strftime("%Y-%m-%d")
+
+
+def scrape_calls(date1: datetime, date2: datetime):
+    url = "https://maps.fayetteville-ar.gov/DispatchLogs/json/getIncidents.cshtml/{}/{}"
+    r = requests.get(url.format(format_date(date1), format_date(date2)))
+    j = r.json()
+    return j
+
+
+def insert_data(calls):
+    with sqlite3.connect(DB_PATH) as conn:
+        c = conn.cursor()
+        for call in calls:
```

Comment:
Hey, so you can insert all data points at once. See [this code snippet](https://github.com/katzrkool/menu/blob/b9e402e30cf2d124cb3e589b7b904a33a5beb469/menu/scraper.py#L104)